documentation/abnormal_termination/README.md

```diff
@@ -39,11 +43,49 @@ To find out if the process exited because of a `process.exit()` call, follow [Us
 
 ### Process exits with Native Stack Trace
 
-TBD
+Some Node.js application uses native modules to re-use existing components or
+performance optimize specific areas. A common example for native modules are
+database drivers. In the case our native module has an issue it can crash the
+Node.js application which will generate a native stack trace.
 
-### Process exits with unmeaningful Stack Trace
+You can find the most common scenarios for native crashes here:
 
-TBD
+1. Native module crashes due bug
+2. Native module crashes due memory leak (native memory leak)
+3. Bugs in Node.js runtime and dependencies
+  - For example: V8 checks, when input checking is missing can lead to crash
```

Comment:
checks in v8 are actually no bugs. missing validation in user code is not a bug in Node.js or it's dependencies.

Reply:
how about?

Operational error in Node.js runtime and native dependencies
  - For example, V8 checks, when input checking is missing can lead to a crash

Reply:
Maybe `For example, V8 checks, when input data checking to v8 APIs checking is missing can lead to a crash`.

Reply:
Changed